portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,16 +24,34 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-    private String me;
+    private ArrayList<String> message;
 
   @Override
   public void init() {
-      me = "Dangely";
+      message = new ArrayList<String>();
+      message.add("Hello Dangely");
+      message.add("This is a test");
+      message.add("Have a nice day!");
   }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello " + me);
+    String json = convertMsgtoJason(message);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  private String convertMsgtoJason(ArrayList<String> msg) {
```

Comment:
convertMsgToJson. Also, you can use Google GSON to do this automatically without having to create your own parser.